README.md

```diff
@@ -29,6 +29,8 @@ Here is my repository link:  https://github.com/chimwemwe007/My-Portfolio
 👤 **Chimwemwe Mkandawire**
 - GitHub: [https://github.com/chimwemwe007/)
 - Twitter: [@CHXMZMK](https://twitter.com/CHXMZMK) 
+- To visit my website please click the following link Portfolio: application deploy
+

```

Comment:
- [ ] I can't find the portfolio live demo link. Kindly add it.

- [ ] Also, this is not where the link should be located. According to the [Microverse template](https://github.com/microverseinc/readme-template), there is a section for listing the live demo. Kindly add it that way.